### DISCUSSION TITLE
hyperparameter tuning

### DISCUSSION THREAD

**Discussion by husai033** [aub-mind/arabert#84] [Q&A]
Thanks for the "Text-classification/Sentiment Analysis with Hyper Parameter tuning" colab. Could you please provide suggestions to solve the following issue:
I got an error message when I run "best_run = trainer.hyperparameter_search(..." as the following:
OperationalError: unable to open database file

The above exception was the direct cause of the following exception:

OperationalError                          Traceback (most recent call last)
/usr/local/lib/python3.7/dist-packages/sqlalchemy/engine/default.py in connect(self, *cargs, **cparams)
    506     def connect(self, *cargs, **cparams):
    507         # inherits the docstring from interfaces.Dialect.connect
--> 508         return self.dbapi.connect(*cargs, **cparams)
    509 
    510     def create_connect_args(self, url):

OperationalError: (sqlite3.OperationalError) unable to open database file
(Background on this error at: http://sqlalche.me/e/13/e3q8)

**Comment by WissamAntoun** [comment]
can you check if you have google drive connected because the SQL storage link point to drive (`  storage="sqlite:////content/drive/MyDrive/{}.db".format(name),`) you can also change it to the local file system

**Reply by husai033** [reply]
Do you mean drive mount? I already mounted it. Could you please explain how to connect it and how to change it to local file? also, does Optuna applies cross-validation approach during hyperparameter tuning? your help is highly appreciated
